.github/workflows/build-image.yml

```diff
@@ -115,6 +115,12 @@ jobs:
           registry: ${{ inputs.regitry_url }}
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Login to ECR
+        uses: docker/login-action@v2
+        with:
+          registry: 819313256810.dkr.ecr.eu-west-1.amazonaws.com
```

Comment:
Il faut que tu variabilise cette url qui change suivant la région et le compte AWS, tu peux prendre exemple sur le workflow `push-image-ghcr-to-ecr.yml`